Fedoraware/TeamFortress2/TeamFortress2/Features/ESP/ESP.cpp

```diff
@@ -580,17 +580,19 @@ void CESP::DrawPlayers(CBaseEntity* pLocal)
 					}
 				}
 			}
-			
-			if (g_EntityCache.IsFriend(nIndex) && Vars::ESP::Players::FriendCond.Value)
-			{
-				size_t FONT = FONT_ESP_COND;
-				int offset = g_Draw.GetFont(FONT).nTall / 4;
-				std::vector<std::wstring> cond_strings = GetPlayerConds(Player);
+			                   if (g_EntityCache.IsFriend(nIndex))
+                                         {
+ 
+
+	                                      const size_t FONT = FONT_ESP_COND;
+                                              const int offset = g_Draw.m_vecFonts[FONT].nTall / 4;
+                                              const std::vector<std::wstring> condStrings = GetPlayerConds(Player); 
+ 
+                                              const wchar_t* friendLabel = L"FRIEND";
+                                              g_Draw.String(FONT, nTextX, y + nTextOffset, Colors::Cond, ALIGN_DEFAULT, friendLabel);
+                                              nTextOffset += g_Draw.m_vecFonts[FONT].nTall;
```

Comment:
This needs to be g_Draw.GetFont(FONT)

Reply:
Oh god. I just noticed that just now. Ty, for telling me.